mlir/include/mlir-c/Dialect/SparseTensor.h

```diff
@@ -25,21 +25,24 @@ MLIR_DECLARE_CAPI_DIALECT_REGISTRATION(SparseTensor, sparse_tensor);
 /// These correspond to SparseTensorEncodingAttr::LevelType in the C++ API.
 /// If updating, keep them in sync and update the static_assert in the impl
 /// file.
+
+typedef uint64_t level_type;
```

Comment:
shouldn't this name be MlirSparseTensorLevelType?

Reply:
To avoid naming conflict with the enum, I used this name. Reasoning for keeping the enum is in the other comment.